app.js

```diff
@@ -341,17 +341,16 @@ const topicExists = function topicExists(id) {
   return docManager.fileExists(fileName, id);
 };
 
-// TODO change to a DELTE /topic/:id
-app.delete('/file', (req, res) => {
+app.delete('/topic/:id', (req, res) => {
```

Comment:
Fair point. I think it makes sense if the content may be deleted by both the creator and the supervisor, but not by an assigned student that is *not* also the creator.